resources/assets/js/app/components/MainNavigation.js

```diff
@@ -29,8 +31,10 @@ class MainNavigation extends Component {
     render() {
         const navClass = !this.state.isNavOpen ? 'collapse' : '';
         const dropdownClass = this.state.isDropdownOpen ? 'show' : '';
-        const logInUrl = !!sessionStorage.jwt ? '/logout' : '/login';
-        const logInMsg = !!sessionStorage.jwt ? 'Logout' : 'Login';
+        const logInUrl = isAuthTokenExists() ? '/logout' : '/login';
```

Comment:
rename method to isAuthorized or hasSession or similar since auth authorization rules can be based not just on JWT token